cmake/checks/check_01_cxx_features.cmake

```diff
@@ -28,29 +29,71 @@
 
 ########################################################################
 #                                                                      #
-#                             C++11 Support:                           #
+#                         C++ Version Support:                         #
 #                                                                      #
 ########################################################################
 
+#
+# backwards compatability with the old DEAL_II_CXX11_FLAG option
```

Comment:
The worst part of that is I just fixed "compatbile" to "compatible" in another file. English is too hard :(